client/src/main/scala/drt/client/components/TerminalDashboardComponent.scala

```diff
@@ -32,7 +32,8 @@ object TerminalDashboardComponent {
                     router: RouterCtl[Loc],
                     portState: PortState,
                     passengerInfoSummary: Map[UtcDate, Map[ArrivalKey, PassengerInfoSummary]],
-                    featureFlags: Pot[Map[String, Boolean]]
+                    featureFlags: Pot[Map[String, Boolean]],
```

Comment:
i might be missing it but are we maybe not using the feature flags pot any more? if so we could take it out, and then take out the feature flags pot..

Reply:
I think I decided to leave it in a while ago because it's such a pain adding and removing it and we quite often add new features we want to flag. Happy to take it out though if you think that's more sensible.